LibreNMS/OS/Ciscoair.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Ios.php
```

Comment:
Can you update the comments to be Ciscoair.

Reply:
Thanks for the information.